pages/data/nsde/_meta.yaml

```diff
@@ -0,0 +1,38 @@
+type: dataset
+documentTitle: 'openFDA › Datasets › MAUDE'
```

Comment:
Fixed. I apparently made the nsde changes in the maude dataset file. Both locations now appropriately reflect their datasets.